onboarding/mentors.md

```diff
@@ -3,4 +3,79 @@ title: Onboarding Responsibilities for Mentors of New Hires
+  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
+    - Point people in Workplace & People Ops
+    - Their manager
+    - Their team lead
+    - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
+  
+
+### Your first meeting
+Get to know each other. This can be a nice, casual conversation. Focus on active listening.
+- Talk about your own history at and before joining Artsy
+- Where are they coming from?
+  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote work?
+  - Are they new in town? Where do they live, for how long? Just have a normal conversation.
+- Prior experience with our technology stacks (and others)
+- Other relevant past experience
+- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+### Your first week
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+- Look for chances to pair or work alongside one another throughout the week.
+  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
+  - If they haven't set up hokusai yet, consider pairing on that.
+- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit things you talked about in the first meeting and look for areas that could have gone better. Consider amending relevant documents like this one.
+
+### Your first year
+- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of 2 months. It's good to check in with your mentee during this time.
+- Consider reaching out to other mentors as well
+- Periodically check in with yourself on what you think you know about your mentee:
+  - How are they feeling at work? It's good to be aware (if you can) of comfort level, new technologies and more dynamic pieces of how they are getting situated at Artsy.
+  - Are there any opportunities to pair together (or for them to pair with someone else) on a new technology or domain area?
+- Many mentoring pairs continue scheduling semi-weekly coffee chats in the weeks and months going forward. As the relationship becomes less formal these can be great to keep in touch with a team member you have a special relationship with!
+
+## Footnotes
+#### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
```

Comment:
> - At Artsy we offer each new team members a spirit guide— a tenured team member who works in a different part of the company and support the onboarding employee’s transition to Artsy.  Who listens, offers advice and helps new teammates feel comfortable in their new work environment. 

From 🔒  [a blog post we never shipped on mentorship](https://docs.google.com/document/d/1LjO8snEL5RB0fKDUd6TqufllJdqyh_dFmibZaXduhV4/edit?usp=sharing) /cc @ashfurrow  who probably also forgot about this

Reply:
I did forget about this, yeah!